### PR TITLE
Bitcoin gas costs

### PR DESCRIPTION
**Pull request by dietersommer**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/integrations/bitcoin/bitcoin-how-it-works.md [hunk 1]
```diff
@@ -97,42 +97,35 @@ Once a smart contract is ready for acceptance testing, it is deployed on the IC
 
 The final stage of development of a Bitcoin smart contract is its deployment on the IC with the Bitcoin API set to use Bitcoin Mainnet. This is the final production environment for the smart contract and is now available.
 
-## API fees
+## API fees / Pricing
```
Comment (THLO) [inline docs/developer-docs/integrations/bitcoin/bitcoin-how-it-works.md:100]:
```suggestion
## API fees & Pricing
```

File: docs/developer-docs/integrations/bitcoin/bitcoin-how-it-works.md [hunk 2]
```diff
@@ -97,42 +97,35 @@ Once a smart contract is ready for acceptance testing, it is deployed on the IC
...
+The costs of API calls in cycles and USD for the Bitcoin Testnet and Bitcoin Mainnet APIs is presented in the following tables. As a general principle for the Bitcoin API, some API calls must have a minimum amount of cycles attached to the call as indicated in the column *Minimum cycles to send with call*. Cycles not consumed by the call are returned to the caller. The reason for providing the minimum cycles is future proofness of the API, i.e., to allow for pricing of API calls to change without breaking existing smart contracts when the Bitcoin subnet grows in terms of replication factor in the future. The call for submitting a Bitcoin transaction to the Bitcoin network does not require one to attach extra cycles as the charged cost is independent of the replication factor of the subnet.
```
Comment (THLO) [inline docs/developer-docs/integrations/bitcoin/bitcoin-how-it-works.md:102]:
```suggestion
The costs of API calls in cycles and USD for the Bitcoin Testnet and Bitcoin Mainnet APIs is presented in the following tables. As a general principle for the Bitcoin API, some API calls must have a minimum amount of cycles attached to the call as indicated in the column *Minimum cycles to send with call*. Cycles not consumed by the call are returned to the caller. Requiring a relatively large minimum number of cycles makes it possible to change the pricing of API calls without breaking existing smart contracts when the Bitcoin subnet grows in terms of replication factor in the future. The call for submitting a Bitcoin transaction to the Bitcoin network does not require extra cycles to be attached as the charged cost is independent of the replication factor of the subnet.
```

**Review by THLO — APPROVED** [review]

**Review by ielashi — APPROVED** [review]

File: docs/developer-docs/gas-cost.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ A thorough example how the cost of running a canister on a 13-node app subnet is
 | HTTPS outcall request                | For sending an HTTPS outcall to a server outside the IC, per message (`http_request`)                            | 400M / 13                             | 400M                                  | 400M / 13 * 34                     |
 | HTTPS outcall payload                | For sending an HTTPS outcall to a server outside the IC, per request and reserved response byte (`http_request`) | 100K / 13                             | 100K                                  | 100K / 13 * 34                     |
 
-Pricing for the Bitcoin API is available in the [Bitcoin API documentation](https://internetcomputer.org/docs/current/developer-docs/integrations/bitcoin/bitcoin-how-it-works).
+Pricing for the Bitcoin API is available in the [Bitcoin API documentation](./integrations/bitcoin/bitcoin-how-it-works).
```
Comment (sesi200) [inline docs/developer-docs/gas-cost.md:46]:
```suggestion
Pricing for the Bitcoin API is available in the [Bitcoin API documentation](./integrations/bitcoin/bitcoin-how-it-works.md).
```

File: docs/developer-docs/gas-cost.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ The Internet Computer is a decentralized public utility, controlled by the NNS 
...
 
-In late November 2022, high-replication application subnets have been made available on the Internet Computer. The first such subnets launched with a replication factor in the order of 30, different sizes may become available in the future. Cycles prices for the new high-replication subnets are scaled linearly based on the number of nodes from the base prices for 13-node subnets. The pricing mechanics for the Bitcoin Mainnet API is slightly different, see the [Bitcoin API documentation](https://internetcomputer.org/docs/current/developer-docs/integrations/bitcoin/bitcoin-how-it-works) for details.
+In late November 2022, high-replication application subnets have been made available on the Internet Computer. The first such subnets launched with a replication factor in the order of 30, different sizes may become available in the future. Cycles prices for the new high-replication subnets are scaled linearly based on the number of nodes from the base prices for 13-node subnets. The pricing mechanics for the Bitcoin Mainnet API is slightly different, see the [Bitcoin API documentation](./integrations/bitcoin/bitcoin-how-it-works) for details.
```
Comment (sesi200) [inline docs/developer-docs/gas-cost.md:15]:
```suggestion
In late November 2022, high-replication application subnets have been made available on the Internet Computer. The first such subnets launched with a replication factor in the order of 30, different sizes may become available in the future. Cycles prices for the new high-replication subnets are scaled linearly based on the number of nodes from the base prices for 13-node subnets. The pricing mechanics for the Bitcoin Mainnet API is slightly different, see the [Bitcoin API documentation](./integrations/bitcoin/bitcoin-how-it-works.md) for details.
```

**Review by sesi200 — COMMENTED** [review]
Needs the extension, otherwise it 404's

**Review by sesi200 — APPROVED** [review]

**Pull request merged**